src/schema/convert-edtr-io-state.tsx

```diff
@@ -10,286 +15,224 @@ import {
-    return [
-      {
-        type: '@edtr-io/exercise',
-        state: {
-          content: convert((node.state as EditorStateDummy).content!),
-          interactive: (convert(
-            (node.state as EditorStateDummy).interactive
-          )[0] as unknown) as ExerciseChildData['state']['interactive'],
-        },
-      },
-    ]
-  }*/
-  // TODO handle this manually in the fetcher!
-  /*if (plugin === 'solution') {
-    return [
-      {
-        type: '@edtr-io/solution',
-        state: {
-          prerequisite: ((node.state as unknown) as SolutionChildData['state'])
-            .prerequisite,
-          strategy: convert((node.state as EditorStateDummy).strategy),
-          steps: convert((node.state as EditorStateDummy).steps),
-        },
-      },
-    ]
-  }
-  if (plugin === 'scMcExercise') {
-    return [
-      {
-        plugin: 'scMcExercise',
-        state: {
-          //@ts-expect-error
-          isSingleChoice: (node.state as EditorStateDummy).isSingleChoice,
-          answers: (node.state as EditorStateDummy).answers.map((answer) => {
-            return {
-              isCorrect: answer.isCorrect,
-              content: convert(answer.content!),
-              feedback: convert(answer.feedback),
-            }
-          }),
-        },
-      },
-    ]
-  }
-  if (plugin === 'inputExercise') {
-    return [
-      {
-        plugin: 'inputExercise',
-        state: {
-          //@ts-expect-error
-          type: (node.state as EditorStateDummy).type,
-          unit: (node.state as EditorStateDummy).unit,
-          answers: (node.state as EditorStateDummy).answers,
-        },
-      },
-    ]
-  }*/
-  if (plugin === 'equations') {
-    const steps = (node.state as EditorStateDummy).steps.map((step) => {
+  if (node.plugin === 'equations') {
+    const steps = node.state.steps.map((step) => {
       return {
         left: convert(step.left),
-        sign: step.sign,
+        // @ts-expect-error
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        sign: step.sign, //TODO: Investigate, probably a bug
         right: convert(step.right),
         transform: convert(step.transform),
       }
     })
     return [{ type: 'equations', steps }]
   }
 
-  const type = node.type
-  if (type === 'p') {
-    // compat unwrap math from p
+  return []
+}
+
+function convertSlate(node: SlateBlockMock) {
+  if (node.type === 'p') {
     const children = convert(node.children)
+
+    // compat unwrap math from p
     if (children.length === 1 && children[0].type === 'math') {
       return children
     }
     // compat: unwrap ul/ol from p
+    //TODO: can not reproduce, does not seem to happen, check again or delete
     if (
       children.length === 1 &&
       (children[0].type === 'ul' || children[0].type === 'ol')
     ) {
       return children
     }
+
     // compat handle newlines
+    //TODO: can not reproduce, what is expected behaviour here?
```

Comment:
https://de.serlo.org/mathe/zahlen-gr%C3%B6%C3%9Fen/grundrechenarten/addition/addition

The first paragraph contains new lines. These are converted into paragraphs.